main/com.spotify/spotifyd.js

```diff
@@ -146,7 +146,7 @@ module.exports = class SpotifyDaemon {
             "--device-name", this.options.device_name,
             "--token", this.options.token,
             "--no-daemon",
-            "--backend", "pulseaudio"
+            "--backend", process.arch.match('arm') ? "alsa" : "pulseaudio"
```

Comment:
Is there a reason why arm needs ALSA instead of PulseAudio? We still need PulseAudio for native sound, and inside the add-on there won't be direct audio HW access.

Reply:
the version of spotifyd we're using for arm is built only with alsa. In the next version (static binary w/musl) we will be customizing also the backend to use pulseaudio.